content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -0,0 +1,116 @@
+---
+html: get-started-evm-sidechain.html
+parent: evm-sidechains.html
+blurb: Get started with the EVM compatible sidechain for the XRP Ledger.
+labels:
+  - Development, Interoperability
+status: not_enabled
+---
+# Get Started with the EVM Sidechain
+
+This getting started tutorial walks you through the steps to set up your account and transfer funds using the EVM sidechain bridge. 
+
+## 1. Create an Account Using an EVM Compatible Wallet
+
+In order to interact with the network, you need to create an account in the EVM sidechain. To create and manage this account you can use any EVM compatible wallet such as MetaMask.
+
+For instructions on how to install and create an account using MetaMask, and then send and receive tokens, see [Connect MetaMask to XRPL Sidechain](connect-metamask-to-xrpl-evm-sidechain.html).
```

Comment:
MetaMask, then

[The _and_ is unnecessary, and there are too many ands already.]